src/CSharp.Workspaces/CSharp/SyntaxLogicalInverter.cs

```diff
@@ -208,6 +208,23 @@ internal static SyntaxLogicalInverter GetInstance(Document document)
                 {
                     return DefaultInvert(expression);
                 }
+            case SyntaxKind.CoalesceExpression:
+            {
+                var binaryExpression = (BinaryExpressionSyntax)expression;
+                if (binaryExpression.Right.Kind() == SyntaxKind.FalseLiteralExpression)
```

Comment:
Please use `IsKind` instead of `Kind`. It it preferred in the codebase. It has also slightly better perf.